README.md

```diff
@@ -1,3 +1,133 @@
 # Fluence Compute Engine
 
 FCE is intended to run various Wasm binaries. At now, it is in the heavily developing phase.
+
+# Installation
+- Clone this project
+- `cargo install --path <path-to-project/tools/cli>`
+
+# Usage
+- `fce build` in Rust project
+
+# HOW TO: Create App with FCE Modules
+
+## Recommendations:
+
+- Modules architecture should be upwards from `effectors` (WASI modules) that will work with local binaries, local storage and syscalls to `pure modules` that perform business logic.
+- Splitting app to small FCE modules are easier to support, reuse and distribute
+- Each module for its own task (npm like)
+
+## Module project structure
+
+- Init simple rust project `cargo init --bin`
+
+- `Config.toml`:
+```
+[[bin]]
+name = "wasm_application"
+path = "src/main.rs"
+
+[dependencies]
+// logger - if you'll use logging
+fluence = { git = "https://github.com/fluencelabs/rust-sdk", features = ["logger"] }
+```
+
+- Methods that will be used outside this module marked with `#[fce]`
+```
+use fluence::fce;
+
+#[fce]
+pub fn get(url: String) -> String {
+...
+}
+```
+- Multiple arguments with primitive Rust types and only one return argument could be used
```

Comment:
I suggest to enumerate all primitive types that could be used here. At now they are
`bool`,`u8`,`u16`,`u32`,`u64`,`i8`,`i16`,`i32`,`i64`,`f32`,`f64`,`String`,`Vec<u8>`